csi/crypto/crypto.go

```diff
@@ -145,6 +145,16 @@ func ResizeEncryptoDevice(volume, passphrase string) error {
 	return err
 }
 
+// IsDeviceOpenAtNullPath determines if encrypted device is already open at unknown device path. The command 'cryptsetup status [device]' show "device:  (null)"
+func IsDeviceOpenAtNullPath(device string) (bool, error) {
```

Comment:
```suggestion
func IsDeviceMappedToNullPath(device string) (bool, error) {
```

In addition, the function only checks `null`. What does `unknown device path` mean in the function description?

Reply:
Modified the method name and description from `unknown device path` to `null device path`. Thanks